.github/workflows/dhis2-verify-app.yml

```diff
@@ -27,7 +27,7 @@ jobs:
             - uses: actions/checkout@v3
             - uses: actions/setup-node@v3
               with:
-                  node-version: 18.x
+                  node-version: 16.x
```

Comment:
Could you just add a comment in the description about why node was downgraded?

Reply:
Thanks for pointing this out. I don't think I deliberately downgraded the node version, but I did want to make sure we were using the same version in all of the different workflow `.yaml` files.

I've now pushed a commit that tries v18 everywhere. If nothing crashes, I'll stick with that.

Reply:
It worked, so sticking with it.